test/app/actions/async.test.js

```diff
@@ -121,9 +113,16 @@ describe('Asynchronous Actions', () => {
           meta: {source: actionSources[actionTypes.SET_NEW_PATIENT_URL]}
         },
         {
-          type: actionTypes.SET_PAGE,
-          payload: {page: pages.LOGIN},
-          meta: {source: actionSources[actionTypes.SET_PAGE]}
+          type: '@@router/CALL_HISTORY_METHOD',
+          payload: {
+            args: [ {
+							pathname: '/login',
```

Comment:
I had an issue with tabs vs spaces in my editor for a moment. Thought I'd found all the places it messed with them and fixed them. Apparently not quite.